WasteManagement/waste-management-context.jsonld

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
does not seem to be in sync with your samples (eg illegalDeposite vs wildDeposite)